sdks/python/apache_beam/testing/datatype_inference.py

```diff
@@ -34,6 +35,9 @@
   from avro.schema import parse as Parse  # avro library for python2
 # pylint: enable=wrong-import-order, wrong-import-position
 
+if not (platform.system() == 'Windows' and sys.version_info[0] == 2):
```

Comment:
@tvalentyn is adding more cases where pyarrow is not supported. I will suggest using the general pattern we use in other places with

try:
  import ... as pa
except ImportError
  pa = None

...

if not pa:
...

Reply:
+1 to @aaltay's suggestion.

Reply:
@udim, @tvalentyn is there an issue for cleaning this up?

Reply:
https://issues.apache.org/jira/browse/BEAM-8188 opened for the cleanup

Reply:
Thank you!